pyEpiabm/docs/source/sweeps.rst

```diff
@@ -9,6 +9,7 @@ Sweep provides the basic class framework to build methods which sweep over the p
 Overview:
 
 - :class:`AbstractSweep`
+- :class:`DemographicsSweep`
```

Comment:
We have tried to prefix sweeps that should only be ran once at the start of the simulation with 'Initial' - I think this is one of them?

Reply:
Yes, this sweep is only ran once

Reply:
Yes, I will change this

Reply:
It looks like you've changed this below, but not in the overview - can you double check?

Reply:
Ah apologies - have fixed that now